docs/en_us/platform_api/Makefile

```diff
@@ -1,164 +0,0 @@
-# Makefile for Sphinx documentation
```

Comment:
@andy-armstrong We have made a decision to mark this API guide deprecated, both within the guide and on docs.edx.org and anywhere else where it makes sense. So we need to be able to update the files and build one more time. @robrap We'll need to revert the deletion of any sphinx/build related files so that we can build.